CLI tools/CurseForge to MultiMC-Git.py

```diff
@@ -35,23 +38,23 @@ def copy_dir(from_path: Path, to_path: Path, from_desc: str, to_desc: str) -> No
 
 def copy_file(from_path: Path, to_path: Path, from_desc: str, to_desc: str) -> None:
     if from_path.is_file():
+        print(f"Copying {from_desc} to {to_desc}")
         shutil.copy2(from_path, to_path)
         print(f"Copied {from_desc} to {to_desc}")
     else:
         print(f"Skipped {from_desc} copying to {to_desc}, didn't exist")
 
 
 # CurseForge to MultiMC
-remove_dir(Path(mmc_path, "mods"), "MultiMC mods")
-remove_dir(Path(mmc_path, "config"), "MultiMC configs")
-remove_dir(Path(mmc_path, "resourcepacks"), "MultiMC resourcepacks")
-remove_file(Path(mmc_path, "options.txt"), "MultiMC options.txt")
-copy_dir(Path(cf_path, "mods"), Path(mmc_path, "mods"), "CurseForge mods", "MultiMC")
-copy_dir(Path(cf_path, "config"), Path(mmc_path, "config"), "CurseForge configs", "MultiMC")
-copy_dir(Path(cf_path, "resourcepacks"), Path(mmc_path, "resourcepacks"), "CurseForge resource packs", "MultiMC")
+remove_dir(mmc_path / "mods", "MultiMC mods")

```

Comment:
If this has `/` then I think ln 9 should lack the last `/` in the path. Similar case for `cf_path` and `git_path` (ln 7).

Reply:
> If this has `/` then I think ln 9 should lack the last `/` in the path. Similar case for `cf_path` and `git_path` (ln 7).

`/` isn't about just adding `/`. It's overloaded `__div__` method of `Path` class with it's own functionality. There's no need to worry about possible incorrect paths when using `pathlib`. It handles everything automatically and won't add unneeded characters.